Zeeguu Reader/Zeeguu Reader/ArticleListViewController.swift

```diff
@@ -116,26 +116,17 @@ class ArticleListViewController: ZGTableViewController {
 	}
 
 	override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-		let queueCell = tableView.dequeueReusableCellWithIdentifier("Cell")
-		var cell: UITableViewCell
+		let queueCell = tableView.dequeueReusableCellWithIdentifier("Cell") as? ArticleTableViewCell
```

Comment:
also, here you seem to try to reuse an existing cell? comment would be cool! thx